docs/data-management/qaqc-criteria.md

```diff
@@ -0,0 +1,196 @@
+# QA/QC criteria
+
+The following lists the pre-defined exclusion criteria for analyses of whole-brain structural and functional connectomes.
+
+## Anatomical MRI
+
+Given our planned analysis, the T1w image will be used for the spatial alignment with the standard MNI152NLin2009cAsym template.
+In addition, surface reconstructions from the T1w image will guide the co-registration of structural and functional (BOLD) images in fMRIPrep.
+Since the latter preprocessing steps are relatively robust to structural images with mild artifacts, the exclusion criteria for unprocessed T1w images are lenient.
+However, individual T1w images may be excluded without such a decision lead to exclusion of the whole session in which it belongs.
+We annotate subjects with visible artifacts in the T1w images in order to ensure rigorous scrutinizing of spatial normalization and surface reconstruction outputs from fMRIPrep (if both modalities passed the first QC checkpoint with MRIQC).
+The explanation and the description of the flagging criteria 1 to 7 are the same as their counterpart in the previous section.
+
+## Summary
+
+#. Artifactual structures in the background
+#. Susceptibility distortion artifacts
+#. Aliasing ghosts
+#. Wrap-around that overlaps with the brain
+#. Data formatting issues
+#. Motion-related and Gibbs ringing
+#. Extreme intensity non-uniformity
+#. Eye spillover
+
+## Artifacts and quality issues that may grant exclusion
+
+### Motion-related and Gibbs ringing
+
+Large head motion during the acquisition of T1w images often expresses itself with the appearance of concentric ripples throughout the scan.
+In the most subtle cases, motion-related ripples look similar to the fine lines generated by Gibbs ringing.
+The latter emerges as a consequence of the truncation of the Fourier series approximation and appears as multiple fine lines immediately adjacent and parallel to high-contrast interfaces.
+While Gibbs ringing is limited to the adjacency of sharp steps in intensity at tissue interfaces, the ripples caused by motion generally span up to the whole brain, and are primarily visible in the sagittal view of MRIQC’s mosaic views.
```

Comment:
Overall, this section should not *try to school* ourselves or others about artifacts etc.

Instead, it should state clear exclusion criteria in a very specific way.

Perhaps we should move this part from the `mriqc.md` file here:

```md
- [ ] Visualize the first mosaic (zoomed into brain mask):
    - [ ] Check for signal *ripples* around the frontal/prefrontal cortex typically caused by head motion
    - [ ] Check for signal interference leaked from the eyeballs across the PE direction overlapping with brain tissue.
    - [ ] Check for ghosts within the brain:
        - [ ] Overlapping wrap-around.
        - [ ] Nyquist aliases (typically through PE direction).
        - [ ] Ghosts caused by external elements such as headsets or mirror frames.
    - [ ] Check for other artifacts such as fat shifts or RF spoiling within the brain.
    - [ ] Check for zipper artifacts and other EM interferences.
    - [ ] Check for excessive B<sub>1</sub> field inhomogeneity.
    - [ ] Check for inhomogeneous *salt-and-pepper* noise patterns.
    - [ ] Check for global *salt-and-pepper* noise distribution.
- [ ] Scroll down to the background-enhanced mosaic:
    - [ ] Check for signal *ripples* around the head typically caused by head motion.
    - [ ] Check for signal interference leaked from the eyeballs across the PE direction.
        If present, go back to the brain mosaic to confirm whether they overlap with the brain tissue.
    - [ ] Check for ghosts outside the brain, and evaluate whether they may overlap with brain tissue:
        - [ ] Overlapping wrap-around.
        - [ ] Nyquist aliases (typically through PE direction).
        - [ ] Ghosts caused by external elements such as headsets or mirror frames.
    - [ ] Check for other artifacts such as RF spoiling or objects outside the head.
    - [ ] Check for zipper artifacts and other EM interferences.
    - [ ] Check for global *salt-and-pepper* noise distribution in the background.
    - [ ] Check for anomalous noise distribution in the background.
```

In `mriqc.md` we just keep:

```md
- [ ] Visualize the first mosaic (zoomed into brain mask) and apply the [exclusion criteria](qaqc-criteria.md#anchor-to-section)
- [ ] Scroll down to the background-enhanced mosaic and apply the [exclusion criteria](qaqc-criteria.md#anchor-to-section)
```

While in `qaqc-criteria.md` we have:

```md
## Anatomical

### Brain mosaic:
- [ ] Check for signal *ripples* around the frontal/prefrontal cortex typically caused by head motion.
    Exclude this particular T1w if ripples are clear and globally localized.
    These T1w images could degrade the quality of surface reconstruction. 
- [ ] Check for signal interference leaked from the eyeballs across the PE direction overlapping with brain tissue.
    Exclude this particular T1w if the leaked signal substantially overlaps cortical brain areas.
    These T1w images could degrade the quality of surface reconstruction. 
- [ ] Check for ghosts within the brain:
      - [ ] Overlapping wrap-around.
      - [ ] Nyquist aliases (typically through PE direction).
      - [ ] Ghosts caused by external elements such as headsets or mirror frames.
      
    Exclude this particular T1w if any of these ghosts overlap cortical gray matter.

- [ ] Check for other artifacts such as fat shifts or RF spoiling within the brain.
   Exclude this particular T1w if any of these ghosts overlap cortical gray matter.
- [ ] Check for zipper artifacts and other EM interferences.
    Exclude this particular T1w if any of these ghosts overlap cortical gray matter.
- [ ] Check for excessive B<sub>1</sub> field inhomogeneity.
    Exclude only if it is evident that a coil failure happened.
- [ ] Check for inhomogeneous *salt-and-pepper* noise patterns.
    Generally, do not exclude this T1w image unless the noise pattern destroys cortical gray matter areas.
- [ ] Check for global *salt-and-pepper* noise distribution.
    Generally, do not exclude this T1w image except evident imaging global failure.

### Background-enhanced mosaic
- [ ] Check for signal *ripples* around the head typically caused by head motion.
    Exclude this T1w only if identifying these ripples leads to revising the decision on the brain mosaic.
- [ ] Check for signal interference leaked from the eyeballs across the PE direction.
    Exclude this T1w only if identifying these leakages leads to revising the decision on the brain mosaic.
- [ ] Check for ghosts outside the brain, and evaluate whether they may overlap with brain tissue:
        - [ ] Overlapping wrap-around.
        - [ ] Nyquist aliases (typically through PE direction).
        - [ ] Ghosts caused by external elements such as headsets or mirror frames.
    Exclude this T1w only if identifying these ghosts leads to revising the decision on the brain mosaic.

... continued ...

```

Reply:
Yes, I like the separation of having the checklist of exclusion criteria in `qaqc-criteria.md` and in `mriqc.md` have one line saying "apply those exclusion criteria".

Ok I understand your intention of being very brief and precise in exclusion criteria statement. But what if me or the reader doesn't know what this artifact is?
Part of the solution is to refer to our paper, but e.g. I don't know what is `salt-and-pepper` noise distribution and we didn't cover this in the paper.

I also had the idea to add figures to illustrate the exclusion criteria, but I guess this fall under the *schooling* approach. As such that's not what we are going for today. Can you confirm?